packages/blade/src/utils/mapProps/mapProps.web.test.ts

```diff
@@ -0,0 +1,51 @@
+import mapProps from './mapProps';
+
+describe('mapProps', () => {
+  it('should work', () => {
```

Comment:
can we have better contextual names for test?

these are simple keys so we can mention something like

`should return correct accessibility attributes for flat accessibility keys`

Reply:
yeah i just skimmed through them will improve these descriptions, thanks!